include/exadg/incompressible_navier_stokes/spatial_discretization/operator_dual_splitting.cpp

```diff
@@ -839,8 +839,7 @@ OperatorDualSplitting<dim, Number>::local_interpolate_velocity_dirichlet_bc_boun
         }
         else if(boundary_type == BoundaryTypeU::DirichletCached)
         {
-          auto bc =
-            this->boundary_descriptor->velocity->dirichlet_cached_bc.find(boundary_id)->second;
+          auto bc = this->boundary_descriptor->velocity->get_dirichlet_cached_data();
```

Comment:
bid is not needed anymore?

Reply:
There is only one data structure for all boundary IDs. This data is then accessed by face, quadrature point, and vectorized array indices (for all boundary IDs belonging to the cached-type BC). Before the present PR, all `FunctionCached` objects (for the different boundary IDs) were pointing to the same data structure. (The background is that we collect all boundaries and setup dealii::RPE once).